index.html

```diff
@@ -1,229 +1,243 @@
-                <div class="des1">
-                   <img src="image/lang.png" alt="">  
-                <ul class="ul44">
-                    <li class="lastli">Javascript</li>
-                    <li class="lastli">Ruby</li>
-                    <li class="lastli">Html</li>
-                    <li class="lastli">css</li>
-                </ul>  </div>
-                    
-                <div class="des2">
-                <img class="h4" src="image/frame.png" alt="">
-                <ul>
-                    <li class="lastli2">Bootstrap</li>
-                    <li class="lastli2">CapyBara</li>
-                    <li class="lastli2">Selenium</li>
-                    <li class="lastli2">RSPec</li>
-                </ul> </div>
-                
-                <div class="des3">
-                <img class="h4" src="image/skills.png" alt="">
-                <ul>
-                    <li class="lastli3">Codepen</li>
-                    <li class="lastli3">Terminal</li>
-                    <li class="lastli3">codekit</li>
-                    <li class="lastli3">Github</li>
-                    <li class="lastli3">Gitlab</li>
-                </ul>
-            </div>
-             
-             </div>
-
-            </section>
-           
-
-
-
-
-
-             
-
-
-
-            </div>
-       
-      
-
-
-        
-     
-
-
-
-
-
-
-    <link rel="stylesheet" href="a.css">
-</body>
-</html>
\ No newline at end of file
+        <h3>
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </h3>
+
+        <button class="btn11">Get My Resume</button>
+
+        <hr class="hr2" />
+        <div class="description">
+          <div class="des1">
+            <img src="image/lang.png" alt="" />
+            <ul class="ul44">
+              <li class="lastli">Javascript</li>
+              <li class="lastli">Ruby</li>
+              <li class="lastli">Html</li>
+              <li class="lastli">css</li>
+            </ul>
+          </div>
+
+          <div class="des2">
+            <img class="h4" src="image/frame.png" alt="" />
+            <ul>
+              <li class="lastli2">Bootstrap</li>
+              <li class="lastli2">CapyBara</li>
+              <li class="lastli2">Selenium</li>
+              <li class="lastli2">RSPec</li>
+            </ul>
+          </div>
+
+          <div class="des3">
+            <img class="h4" src="image/skills.png" alt="" />
+            <ul>
+              <li class="lastli3">Codepen</li>
+              <li class="lastli3">Terminal</li>
+              <li class="lastli3">codekit</li>
+              <li class="lastli3">Github</li>
+              <li class="lastli3">Gitlab</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <form
```

Comment:
- [ ] The `contact form` is part of the `contact-me section` which itself is part of the `footer`. Therefore, the semantically meaningful ordering should be something like this:
```
<footer>
     <section class="contact-me">
          <div>
              <h2> Contact Me </h2>
              <p> I'm always interested in hearing .... </p>
          </div>
          <form>
              <input name="fullname" />
              <input name="email" />
              <textarea placehoder="enter message" />
              <input type="submit" />
         </form>
    </section>
</footer>
```

- [ ] You should Flexbox for form element layout as stated in the project requirement. `In order to lay out the elements on the page you should use Flexbox.`